README.rst

```diff
@@ -67,6 +67,9 @@ E.g. to set your master name when using Redis Sentinel for broker discovery:
 Use ``--tz`` to specify the timezone the Celery app is using. Otherwise the
 systems local time will be used.
 
+Use ``--queue_list`` to specify the list of queues that will have its length
```

Comment:
Looks like a typo. It should probably be `--queue-list`? 🙂

Reply:
Yeah, thats right. Fixes coming.